docs/bugpattern/CheckedExceptionNotThrown.md

```diff
@@ -1,11 +1,11 @@
-Java allows methods to declare that they throw checked exceptions even when they
-don't. This can lead to call sites being forced to explicitly handle or
+Java allows methods to declare that they throw checked exceptions even when they don’t.
+This can lead to call sites being forced to explicitly handle or
 propagate exceptions which provably can never occur. It may also lead readers of
 the code to search for possibly throwing paths where none exist.
 
 ```java
 private static void validateRequest(Request request) throws IOException {
-  checkArgument(request.hasFoo(), "foo must be specified");
+  System.out.println("Request " + request.name + " validated");
```

Comment:
That doesn't really look like a "validateRequest" implementation. Maybe replace checkArgument with the equivalent non-Guava code instead if you think it's unclear?
```suggestion
  if (!request.hasFoo()) {
    throw new IllegalArgumentException("foo must be specified");
  }
```